lib/Runtime/Base/FunctionBody.cpp

```diff
@@ -9286,7 +9286,15 @@ namespace Js
                 else
                 {
                     newEntryPoint = functionBody->CreateNewDefaultEntryPoint();
```

Comment:
should we put this in an else statement? (i.e., not create two new entry points in this scope)

Reply:
`CheckCodeGenThunk` is basically a wrapper thunk which will call out to `functionBody->originalEntryPoint` after doing its thing, so we need to get originalEntryPoint set up. Also, GenerateFunction doesn't create a new EntryPointInfo, it just modifies the existing one.

This means that my `newEntryPoint = functionBody->GetDefaultFunctionEntryPointInfo();` line is unnecessary though, so I will remove it.